src/Utilities/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataSymbolMapExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
+{
+    internal static class TaintedDataSymbolMapExtensions
```

Comment:
>TaintedDataSymbolMapExtensions [](start = 26, length = 30)

You can probably remove this type completely if `GetInfosForType` is converted to `GetTaintedSymbolsForType`

Reply:
I like the method names here.